K2Bridge/Visitors/QueryStringClauseVisitor.cs

```diff
@@ -44,10 +44,17 @@ public void Visit(QueryStringClause queryStringClause)
             switch (queryStringClause.ParsedType)
             {
                 case QueryStringClause.Subtype.Term:
-                    var isNumeric = GetIsFieldNumeric(queryStringClause.ParsedFieldName).Result;
-                    if (isNumeric)
+                    var isMatchPhraseFieldType = IsMatchPhraseFieldType(queryStringClause.ParsedFieldName).Result;
```

Comment:
This area will conflict with PR: https://github.com/microsoft/K2Bridge/pull/12
I am marking with do not merge so we will not enter this before.